pmd-kotlin/src/main/resources/rulesets/kotlin/rulesets.properties

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
do we need this file? It would be needed as soon as we provide a quickstart.xml ruleset...